sdf.md

```diff
@@ -312,15 +313,16 @@ are often, but not always, called *qualities*.
 ## Elements of an SDF model
 
 The SDF language uses six predefined Class Name Keywords for modeling connected
-Things which are illustrated in {{fig-class-2}}.
+Things which are illustrated in {{fig-class-2}} (the sixth
+class `sdfObject` is similar enough to `sdfThing` that it would look
+the same in the figure).
 
 ~~~ plantuml-utxt
-sdfThing --> "0+" sdfObject : hasObject
 sdfThing --> "0+" sdfThing : hasThing
 
-sdfObject --> "0+" sdfProperty : hasProperty
-sdfObject --> "0+" sdfAction : hasAction
-sdfObject --> "0+" sdfEvent : hasEvent
+sdfThing --> "0+" sdfProperty : hasProperty
+sdfThing --> "0+" sdfAction : hasAction
+sdfThing --> "0+" sdfEvent : hasEvent
```

Comment:
Should both `sdfThing` and `sdfObject` show up in the figure? If not, should `sdfObject` the one that is being displayed, while `sdfThing` is declared equivalent? Or rather the other way around?